frontend/src/Attribution/Plots.elm

```diff
@@ -146,12 +153,31 @@ plotNgramsSpi { ngramsSpi } =
             |> viewBars (groups (List.map (uncurry group)))
 
 
+plotSimilarities : Statistics -> Html.Html msg
+plotSimilarities { similarity } =
+    let
+        rename : String -> String
+        rename str =
+            str
+                |> Regex.replace Regex.All (Regex.regex "\"") (\_ -> "")
+                |> Regex.replace Regex.All (Regex.regex "_") (\_ -> " ")
+
+        construct ( key, value ) =
+            ( rename (toString key), [ value ] )
```

Comment:
this can be slightly simpler, and needs a comment.

The key is already a string, like "foo" (calling `toString "foo"` produces `"\"foo\""`), so we don't need the toString

you could add a comment saying something like "create user-friendly names for the bars"